server/src/main/java/io/seata/server/lock/DefaultLockManager.java

```diff
@@ -61,6 +62,8 @@ public boolean acquireLock(BranchSession branchSession) throws TransactionExcept
             //no lock
             return true;
         }
+        //sort locks of branch by pk, if this influence performance, could move this work to RM
+        locks.sort(Comparator.comparing(RowLock::getPk));
```

Comment:
Revoke the changes here. Because of the large-scale batch insertion scenario, this is a performance loss that does not actually lock conflicts.

Reply:
OK